book/src/README.md

```diff
@@ -0,0 +1,84 @@
+<img src="./images/amethyst_thumb.png" alt="Logo" width="96px" style="float:left;margin-right:15px"/>
+
+# The Amethyst Engine
+
+> Note: This project is a *work in progress* and is very incomplete. Pardon the
+> dust!
+
+Howdy! This book will teach you everything you need to know about building video
+games and interactive simulations with the Amethyst game engine. This engine is
+written entirely in [Rust][rs], a safe and fast systems programming language,
+and sports a clean and modern design. More correctly, though, Amethyst is
+actually a suite of separate libraries and tools that collectively make up a
+game engine.
+
+[rs]: https://www.rust-lang.org/
+
+Amethyst is free and open source software, distributed under a dual license of [MIT][ml]
+and [Apache][al]. This means that the engine is given to you at no cost 
+and its source code is completely yours to tinker with. The code is available on
+[GitHub][am]. Contributions and feature requests are welcome!
+
+[ml]: https://github.com/amethyst/amethyst/blob/master/LICENSE-MIT
+[al]: https://github.com/amethyst/amethyst/blob/master/LICENSE-APACHE
+[am]: https://github.com/amethyst/amethyst
+
+This book is split into seven sections. This page is the first. The others are:
+
+* [Getting Started][gs] – Prepare your computer for Amethyst development.
+* [A Simple Application][sa] – Build a basic pong game in Rust.
+* Effective Amethyst – Learn how to write complex games and applications.
+* Scripting – Jump into scripted development with [Ruby][rb].
+* [Glossary][gl] - Defines special terms used throughout the book.
+* Bibliography – Read the works that influenced the design.
+
+[gs]: ./getting_started.html
+[sa]: ./simple_application.html
+[rb]: https://www.ruby-lang.org/
+[gl]: ./glossary.html
+
+Read the crate-level [API documentation][ad] for more details.
+
+[ad]: https://www.amethyst.rs/doc/master/amethyst/
+
+## Why are you building this?
```

Comment:
I'm open to suggestions, or maybe leave it as is for now and change it in a separate PR?